examples/react/css/src/variables.css

```diff
@@ -25,6 +25,7 @@
   --ais-spacing-factor: 1;
   --ais-spacing: calc(var(--ais-base-unit) * var(--ais-spacing-factor) * 1px);
   --ais-spacing-half: calc(var(--ais-spacing) / 2);
+  --ais-spacing-third: calc(var(--ais-spacing) / 3);
```

Comment:
I think we decided to stick with `--ais-spacing` only based on @sarahdayan's PR: https://github.com/algolia/instantsearch/pull/6586/files#diff-268c7ac17cc62a4721d38ee84477fa90adb65aba77242d7fffb1c75d6a08488a